template/config.sls

```diff
@@ -1,16 +1,17 @@
 # -*- coding: utf-8 -*-
 # vim: ft=sls
 
-{% from "template/map.jinja" import template with context %}
-{% from "template/macros.jinja" import files_switch with context %}
+{%- from "template/map.jinja" import template with context %}
+{%- from "template/macros.jinja" import files_switch with context %}
 
 include:
   - template.install
 
 template-config:
   file.managed:
     - name: {{ template.config }}
-    - source: {{ files_switch('template', ['example.tmpl'] }}
+    - source: {{ files_switch(['example.tmpl', 'example.tmpl.jinja']) }}
     - mode: 644
     - user: root
     - group: root
+    - template: jinja
```

Comment:
Using a `jinja` template in order to show the `source` of the output file for debugging purposes.